src/node_jsvmapi.h

```diff
@@ -35,6 +36,15 @@
 #endif
 
 namespace node {
+
+struct node_module_old {
```

Comment:
I think it should be able to be in  node_jsvmapi_internal.h.  The only place the external definition should have been necessary was when building against 0.10.x and in that case it would not have been the "old" version but the version in node.h.